actioncable/lib/rails/generators/channel/templates/assets/channel.js

```diff
@@ -0,0 +1,15 @@
+(function() {
+  App.<%= class_name.underscore %> = App.cable.subscriptions.create("<%= class_name %>Channel", {
+    connected: function() {
+      // Called when the subscription is ready for use on the server
+    },
+
+    disconnected: function() {
+      // Called when the subscription has been terminated by the server
+    },
+    received: function(data) {
+      // alled when there's incoming data on the websocket for this channel
+    }
```

Comment:
Yup. Added now.

Here is the output for 

```sh
rails g channel Chat speak
``` 

https://gist.github.com/prathamesh-sonpatki/97e1718f806b92441e66cd8bb2897e14